Makefile

```diff
@@ -1,43 +1,52 @@
-DESTDIR ?= /
-SBINDIR = $(DESTDIR)/usr/sbin
-SYSCONFDIR = $(DESTDIR)/etc
-MANDIR = $(DESTDIR)/usr/share/man
-LOGDIR = $(DESTDIR)/var/log/recap
-CRONDIR = $(DESTDIR)/etc/cron.d
+DESTDIR       ?=
+BINPATH       ?= /sbin
+PREFIX        ?= /usr/local
+BINDIR        ?= $(PREFIX)$(BINPATH)
+MANDIR        ?= $(PREFIX)/share/man
+DOCDIR        ?= $(PREFIX)/share/doc
+SYSCONFDIR    ?= /etc
+CRONDIR       ?= $(SYSCONFDIR)/cron.d
+LOGDIR        ?= /var/log
 
 all:
 	@echo "Nothing to compile, run make install."
 
 install:
 	@echo "Installing scripts..."
-	@install -Dm0755 src/recap $(SBINDIR)/recap
-	@install -Dm0755 src/recaplog $(SBINDIR)/recaplog
-	@install -Dm0755 src/recaptool $(SBINDIR)/recaptool
+	@install -Dm0755 src/recap $(DESTDIR)$(BINDIR)/recap
+	@install -Dm0755 src/recaplog $(DESTDIR)$(BINDIR)/recaplog
+	@install -Dm0755 src/recaptool $(DESTDIR)$(BINDIR)/recaptool
 	@echo "Installing man pages..."
-	@install -Dm0644 src/recap.5 $(MANDIR)/man5/recap.5
-	@install -Dm0644 src/recap.8 $(MANDIR)/man8/recap.8
-	@install -Dm0644 src/recaplog.8 $(MANDIR)/man8/recaplog.8
+	@install -Dm0644 src/recap.5 $(DESTDIR)$(MANDIR)/man5/recap.5
+	@install -Dm0644 src/recap.8 $(DESTDIR)$(MANDIR)/man8/recap.8
+	@install -Dm0644 src/recaplog.8 $(DESTDIR)$(MANDIR)/man8/recaplog.8
+	@install -Dm0644 src/recaptool.8 $(DESTDIR)$(MANDIR)/man8/recaptool.8
 	@echo "Installing configuration..."
-	@install -Dm0644 src/recap.conf $(SYSCONFDIR)/recap
```

Comment:
So yes, as @thebwt points out is what's used on recap.  I chase whether or not this is a good practice or not and could've swear I found a source that would make a distinction on that matter, something like this (from what I remember):

- /etc/__project__  - OK when there is no other config file
- /etc/__project.conf__ - OK when there is a directory file for that project and requires more configs, e.g. /etc/__project__/
- /etc/__project__/ - From above, when needed to include more configs
- /etc/__project.d__/ - Similar as above but more explicit, when needed to include more configs

Don't remember if it was a best practice for packaging or something similar but that made me think it was not completely wrong.  But truth is I don't find at the moment anything to support that and since I don't see any reference to this in the [FSH for /etc/](http://www.pathname.com/fhs/pub/fhs-2.3.html#ETCHOSTSPECIFICSYSTEMCONFIGURATION)  this opens the question to.  What's the right thing to do?

For now is clear that without a new release a change like this may not be recommended, but definitely something to take into consideration.